### PR TITLE
[melodic] Gazebo Dependency and GetModel Fixes

### PR DESCRIPTION
**Pull request by vatanaksoytezer**

### CODE REVIEW

**Comment by vatanaksoytezer** [comment]
@hello-binit fixes https://forum.hello-robot.com/t/error-when-setting-up-stretch-simulation-on-gazebo/213

File: stretch_gazebo/package.xml [hunk 1]
```diff
@@ -15,15 +15,18 @@
...
   <depend>gazebo_msgs</depend>
   <depend>gazebo_plugins</depend>
+  <depend>gazebo_ros_control</depend>
   <depend>gazebo_ros</depend>
   <depend>nav_msgs</depend>
   <depend>realsense_gazebo_plugin</depend>
   <depend>realsense2_camera</depend>
   <depend>realsense2_description</depend>
   <depend>robot_state_publisher</depend>
+  <depend>ros_controllers</depend>
```
Comment (hello-sanchez) [inline stretch_gazebo/package.xml:25]: Adding the dependency removed previous errors.  :)

**Comment by vatanaksoytezer** [comment]
@hello-sanchez No p gain is expected as we use internal Gazebo controller gains. You can safely ignore them. This is a normal behaviour and a design choice in simulation.

**Comment by hello-binit** [comment]
Thanks @vatanaksoytezer!

**Pull request merged**